src/helpers.js

```diff
@@ -117,25 +119,88 @@ export function isTraversable(tileID, me) {
   }
 }
 
+/* eslint no-param-reassign: ["error", { "ignorePropertyModificationsFor": ["bigGrid"] }] */
+
+/*
+ * place all values from smallGrid into bigGrid. Align the upper left corner at x, y
+ */
+export function fillGridWithSubgrid(bigGrid, smallGrid, x, y) {
```

Comment:
I agree, though for this function, it is generalizable to any grids. Perhaps grid should be synonymous with "2d array". Under that assumption, this usage is fine, but others may not be (would have to go through and look). Perhaps we open that as an issue on our backlog?